source/content/guides/launch/03-domains.md

```diff
@@ -86,12 +86,14 @@ Once you begin this process, you have:
 
 1. Serve the file from your existing live site. Drupal 7 users can use the [Let's Encrypt Challenge](https://www.drupal.org/project/letsencrypt_challenge) module to easily serve the contents of the challenge file.
 
-   <Alert title="Note" type="info">
-
-   The validation file to pre-provision HTTPS must be accessible over HTTP, not just HTTPS. A redirect from HTTP to HTTPS will work, but if a request over HTTP returns a 404, for example, the validation will fail.
-
-   If you're unable to host the challenge file, consider using the [Terminus ACME Plugin](https://github.com/pantheon-systems/terminus-acme-plugin) to generate DNS TXT records to validate domain ownership.
+    <Alert title="Note" type="info">
 
+    The validation file to pre-provision HTTPS must be accessible over HTTP, not just HTTPS. A redirect from HTTP to HTTPS will work, but if a request over HTTP returns a 404, for example, the validation will fail.
+    
+    It is recommeneded that you use the Dashboard to get the DNS record instead of the Terminus plugin. A misconfiguration of the validation file invalidates these records and the Dashboard does not provide any feedback. 
```

Comment:
> It is recommeneded that you use the Dashboard to get the DNS record instead of the Terminus plugin.

It looks like "recommeneded" is misspelled. But also, I think this whole sentence should be removed. This is the only place on the page that I see a Terminus plugin mentioned. If we're going to mention it, we should link to it. But I think it just shouldn't be mentioned.